app/views/movies/show.html.erb

```diff
@@ -0,0 +1,76 @@
+  
+<div class="title-block">
+  <div class="row">
+    <div class="col s4 center-align"><h3><%=@movie.title%></h3></div>
+  </div>
+  <div class="row">
+    <div class="col s1 offset-s1 right-align"><p>Rated: <%=@movie.data(:rated)%></p></div>
+    <div class="col s1"><p>Runtime: <%=@movie.data(:runtime)%></p></div>
```

Comment:
You are calling `Movie#data` multiple times for the same movie, even though this call returns the same exact value each time (and makes the exact same HTTP request each time!).  To avoid making these redundant HTTP requests, you could `memoize` your calls to `Movie#data`, so that you are not making multiple HTTP requests for the same data.  Check [this out](http://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/) for how to accomplish this inside `Movie#data`.